templates/bot/js/notification-function-base/README.md

```diff
@@ -1,43 +1,37 @@
-# Send notifications to Teams
+# Overview of the Notification bot template
 
-The Notification in Teams feature enables you to consume, transform  and post events as plain text or [adaptive cards](https://docs.microsoft.com/microsoftteams/platform/task-modules-and-cards/cards/cards-reference) to an individual, chat, or channel in Teams.
+This template showcases an app that send a message to Teams with Adaptive Cards triggered by a HTTP post request. You can further extend the template to consume, transform and post events to individual, chat or channel in Teams.
 
-This app is built with the [Microsoft Bot Framework](https://dev.botframework.com/) running on the Azure Function service along with the [Azure Bot Service](https://azure.microsoft.com/services/bot-services/).
+The app template is built using the TeamsFx SDK, which provides a simple set of functions over the Microsoft Bot Framework to implement this scenario.
 
-Here is a screen shot of the app running:
-
-![Notification Message in Teams](https://user-images.githubusercontent.com/11220663/166959087-a13abe67-e18a-4979-ab29-a8d7663b3489.png)
-
-# Getting Started
-
-Run your app with local debugging by pressing `F5` in VSCode. Select `Debug (Edge)` or `Debug (Chrome)`.
-
-**Congratulations**! You are running an application that can now send notifications to Teams.
-
-## Test a http trigger
-
-If you selected `http` trigger, you can test it:
-
-* Send a POST request to `http://<endpoint>/api/notification` with your favorite tool (like `Postman`)
-  * When your project is running locally, replace `<endpoint>` with `localhost:3978`
-  * When your project is deployed to Azure Functions, replace `<endpoint>` with the url from Azure Functions
+## Get Started with the Notification bot
 
 >
 > **Prerequisites**
 >
-> To run locally, you will need:
+> To run the notification bot template in your local dev machine, you will need:
 >
 > - `Node.js` installed locally (recommended version: 14)
```

Comment:
I believe now we recommend node 16, so better to update this as well.

Reply:
Updated